tools/mo/openvino/tools/mo/convert_impl.py

```diff
@@ -47,8 +47,7 @@
 from openvino.tools.mo.utils.utils import refer_to_faq_msg
 from openvino.tools.mo.utils.telemetry_utils import send_params_info, send_framework_info, send_conversion_result, \
     get_tid
-from openvino.tools.mo.utils.versions_checker import check_requirements, log_not_satisfied_dependencies, \
-    get_environment_setup  # pylint: disable=no-name-in-module
+from openvino.tools.mo.utils.versions_checker import get_environment_setup  # pylint: disable=no-name-in-module
```

Comment:
@popovaan, please explain the reason we still need to use this method. I see below
`
def check_model_object(argv):
    model = argv['input_model']
    if 'tensorflow' in sys.modules:
        import tensorflow as tf
        env_setup = get_environment_setup("tf")
`
Does it mean we will generate a warning when tensorflow is installed and the version doesn't match the expected one? Does it print warning even if a new FE is used?